bin/check-swap-percent.rb

```diff
@@ -44,6 +44,12 @@ def run
     total_swap = output_swap.split(' ')[1].to_i
     used_swap = output_swap.split(' ')[2].to_i
 
+    # Check if swap exists
+    if total_swap.zero?
+      print 'There is no SWAP on this machine'
+      exit
```

Comment:
it can be optional (and none of my vms use it) but the check is specifically checking for swap so it makes sense that someone might care about swap not being present. it could be a bool value with a default to ignore.